tests/elections/test_city_of_london_local.py

```diff
@@ -29,3 +35,101 @@ def test_city_of_london_registration_deadline(election):
         CityOfLondonLocalElection(election["poll_date"]).registration_deadline
         == election["expected_registration_deadline"]
     )
+
+
+sopn_test_cases = [
+    {
+        # https://candidates.democracyclub.org.uk/elections/local.city-of-london.cordwainer.by.2022-09-15/sopn/
+        "poll_date": date(2022, 9, 15),
+        "sopn_publish_date": date(2022, 8, 23),
+    },
+    {
+        # https://candidates.democracyclub.org.uk/elections/local.city-of-london.bassishaw.by.2019-04-30/sopn/
+        # includes an easter break
```

Comment:
I wasn't able to find a real-life example of a City of London election with polling day in late December/early Jan to check the Xmas break case but we did have one including the Easter break.